src/frontend/src/tests/btc/components/convert/BtcConvertTokenWizard.spec.ts

```diff
@@ -0,0 +1,193 @@
+import BtcConvertTokenWizard from '$btc/components/convert/BtcConvertTokenWizard.svelte';
+import * as btcSendServices from '$btc/services/btc-send.services';
```

Comment:
I would say a bit complicated - the number of mocks would only grow if I do so (e.g. instead of mocking sendBtc service, I would need to do so for sendBtcApi, addPendingBtcTransaction, and probably waitAndTriggerWallet). In any case, we won't be able to get the "real" result of a service call (which ideally is unit-tested separately), so I would probably avoid the complexity.

Reply:
I'm hesitant. I definitely understand your point. On the other, I'm a bit afraid that the complexity and coverage grows but, we actually miss some edge case because we do mock at a too high level. How complicated it will be to setup E2E tests for this conversion? If we will be able to have E2E test, I would say it's fine to mock the service. If we will never have E2E test for it, I would say it would be better to not mock the service. WDYT?

Reply:
Totally makes sense. It will definitely take some time until we have e2e tests here, could be quite tricky to somehow mock different APIs to be able to use this and all other flows "natively". So, for now, let's indeed move one layer down and do not touch the service in the unit tests. I am now mocking the underlying APIs. Please take a look.

Reply:
Thanks!